Dockerfile

```diff
@@ -51,3 +51,5 @@ ADD crontab /etc/cron.d/crontab
 RUN chmod 0644 /etc/cron.d/crontab
 RUN touch /var/log/cron.log
 RUN crontab /etc/cron.d/crontab
+
+CMD service cron stop
```

Comment:
This means that cron is running for no image, right? And it is a bit of a hack, methinks. I suppose we could determine if cron needs starting/stopping in this file : https://github.com/metabrainz/listenbrainz-server/blob/master/docker/beta/uwsgi/uwsgi.service (as well as the prod version of it).

Also, changes to this Dockerfile, if we keep it should be made to Dockerfile.prod

Reply:
Seemed hacky to me too. Changing the uwsgi.service script then.

>Also, changes to this Dockerfile, if we keep it should be made to Dockerfile.prod

on it.